pennylane_lightning/lightning_qubit/lightning_qubit.py

```diff
@@ -542,7 +542,10 @@ def c_dtype(self):
         """State vector complex data type."""
         return self._c_dtype
 
-    dtype = c_dtype
+    @property
+    def dtype(self):
+        """State vector complex data type."""
+        return self._c_dtype
```

Comment:
Wasn't the idea behind the previous implementation about not implementing any setter/getter for `dtype`?
```suggestion
    dtype = c_dtype
```
On another side, `c_dtype` is a "property" of the device.

Reply:
Ok, I will change it. I was not sure why `dtype` was not a property :sweat_smile: